libs/events/events_test.go

```diff
@@ -162,6 +166,52 @@ func TestAddDifferentListenerForDifferentEvents(t *testing.T) {
 	}
 }
 
+var stopInputEvent = false
```

Comment:
can we not introduce global variables?

Reply:
I'm sorry for dalayed reply，I'm taking a holiday, so the next relay may also delay.
Yes，this global variables  will eliminated.